compose/src/main/java/com/maplibre/compose/symbols/StyleEx.kt

```diff
@@ -0,0 +1,26 @@
+package com.maplibre.compose.symbols
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.graphics.drawable.VectorDrawable
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.ImageBitmap
+import androidx.compose.ui.graphics.asAndroidBitmap
+import androidx.compose.ui.res.imageResource
+import com.mapbox.mapboxsdk.maps.Style
+
+@SuppressLint("ComposableNaming")
+@Composable
+internal fun Style.addImageFromResourceId(context: Context, resourceId: Int?) {
```

Comment:
@Archdoog @ianthetechie I paired with Nash on best practices here and this is what we came up with. Let me know if any feedback!